awx/main/apps.py

```diff
@@ -63,3 +64,18 @@ def ready(self):
 
         self.load_credential_types_feature()
         self.load_named_url_feature()
+
+    def load_cloud_provider_plugin_names(self):
+        from awx.main.models.inventory import InventorySourceOptions
```

Comment:
Is it possible to avoid having in-function imports?

Reply:
I moved the import to the top of the file in this commit `462bfc7b372c92cbaad09b1f962dd3cffbdad259` Please let me know if that's not what you meant!

Reply:
Ack. It looks like I don't actually have a `Resolve` button in this repo. So you'll need to click it yourself...